### ISSUE TITLE
Screen Readers: About and Subsections: The image of QR code is not focused and announced

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77387]
## Action Performed:
Precondition: The User should be logged in

1. Using Android TalkBack or iOS VoiceOver, open the Expensify app
2. Swipe to the 'Account' button at the bottom right corner of the screen and double-tap to activate
3. Swipe to the 'About' button and double-tap to activate
4. Swipe to the 'App download links' button and double-tap to activate
5. Swipe and try to focus on the image of the QR code

## Expected Result:
The image of QR de should be focused and announced, e.g.:
- 'QR code'

## Actual Result:
The image of QR code is not focused and announced.

## Area issue was found in:
Preferences and  Subsections

## Failed WCAG checkpoints
WCAG 1.1.1 Non-Text Content
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html

## User impact:
Users who have difficulty perceiving visual content use assistive technology to read text aloud, present it visually, or convert it to braille. Text alternatives may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. People who are deaf-blind can read the text alternatives in braille.

## Suggested resolution:
Ensure the image element is correctly marked as accessible and provided with a clear accessibilityLabel (e.g.: 'QR code')
https://reactnative.dev/docs/accessibility#accessible
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment(s)
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/02d671db-7326-4f74-a808-ac4f34742028)

https://github.com/user-attachments/assets/3a0dddc0-3219-44da-9ba6-db9a5a7f49e7

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7006375
**Issue reported by:** NA
**Slack conversation:** NA
